### PR TITLE
Add n9 rules

### PR DESCRIPTION
**Pull request by BenjD90**
based on the PR #2

### CODE REVIEW

File: tslint.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - true
+    - array
+  typedef:
+    - true
+    - call-signature
+    - parameter
+    - member-variable-declaration
+  cyclomatic-complexity:
+    - true
+    - 20
```
Comment (victorboissiere) [inline tslint.yaml:48]: 20 ? seems way too large

Comment (BenjD90) [reply tslint.yaml:48]: ok, I propose 11 :
https://www.elsewebdevelopment.com/typescript-airbnb-style-guide-optimized-for-prettier-in-one-json-file/

Comment (BenjD90) [reply tslint.yaml:48]: default in eslint is 20 : https://eslint.org/docs/rules/complexity

Comment (victorboissiere) [reply tslint.yaml:48]: 11 is better I think ;)

Comment (BenjD90) [reply tslint.yaml]: @victorboissiere I updated to 11 :)

**Pull request merged**